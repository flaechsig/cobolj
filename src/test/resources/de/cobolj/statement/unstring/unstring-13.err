Einfache Unstring-Anweisung mit NOT OVERFLOW 