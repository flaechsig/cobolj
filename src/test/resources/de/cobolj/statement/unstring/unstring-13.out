NO OVERFLOW
00001
Max  
Muster 
00001Max  Muster #
